src/templates/partials/styles.mjml

```diff
@@ -1,22 +1,42 @@
 <mj-attributes>
-      <mj-all font-family="Roboto, Helvetica, Arial, sans-serif" color="#55575d" vertical-align="middle" text-align="center" background-color="#F4F4F4" />
-    </mj-attributes>
-    <mj-style inline="inline">
-      .container {
-        max-width: 600px;
-      }
+  <mj-all font-family="Roboto, Helvetica, Arial, sans-serif" color="#55575d" vertical-align="middle" text-align="center" background-color="#F4F4F4" />
+</mj-attributes>
+<mj-style inline="inline">
+  .container {
+    max-width: 600px;
+  }
 
-      .link {
-        font-family: Roboto Helvetica, Arial, sans-serif;
-        text-decoration: none;
-        font-size: 16px;
-        color: #346df1;
-      }
+  .link {
+    font-family: Roboto Helvetica, Arial, sans-serif;
+    text-decoration: none;
+    font-size: 16px;
+    color: #346df1;
+  }
 
-      .footerLink {
-        font-size: 14px;
-        color: #000000;
-        text-decoration: none;
-        color: #346df1;
-      }
-    </mj-style>
+  .footerLink {
+    font-size: 14px;
+    color: #000000;
+    text-decoration: none;
+    color: #346df1;
+  }
+
+  .weekly ul {
```

Comment:
@balexander82 I don't have a problem with putting any needed styles in here since this is only used in emails and shouldn't really matter if we have extra styles that aren't needed.  That being said, there is always the possibility that these styles will affect other emails where they are not needed, so just keep that in mind when deciding to put styles here or in the email themselves (since this partial was meant for global styles used in multiple emails, not every styles a single email might need).